recipe/bld.bat

```diff
@@ -15,19 +19,19 @@ cmake -G "Ninja" ^
     -DLLVM_TARGETS_TO_BUILD=X86 ^
     -DLLVM_TEMPORARILY_ALLOW_OLD_TOOLCHAIN=ON ^
     -DLLDB_ENABLE_PYTHON=ON ^
-    -DLLDB_PYTHON_RELATIVE_PATH=../Lib/site-packages ^
-    -DPYTHON_HOME=%PREFIX% ^
+    "-DLLDB_PYTHON_RELATIVE_PATH:PATH=..\Lib\site-packages" ^
```

Comment:
Style nit
```suggestion
    -DLLDB_PYTHON_RELATIVE_PATH:PATH="..\Lib\site-packages" ^
```

Reply:
Let's hope this doesn't make a difference (it is actually something differently on Windows)